src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -114,17 +119,10 @@ namespace NServiceBus
     {
         public static void TimeToWaitBeforeTriggeringCriticalErrorOnTimeoutOutages(this NServiceBus.BusConfiguration config, System.TimeSpan timeToWait) { }
     }
+    [System.ObsoleteAttribute("This is no longer a public API. Will be removed in version 7.0.0.", true)]
```

Comment:
Can you explain why you obsoleted it? Especially `TypesToScan` could be interesting for users